internal/cmd/cmd_test.go

```diff
@@ -35,6 +36,12 @@ func TestCanonicalPath(t *testing.T) {
 	t.Setenv("HOME", "/home/user")
 	t.Setenv("USERPROFILE", "/home/user")
 	wd := t.TempDir()
+
+	if runtime.GOOS == "darwin" {
```

Comment:
Not the most elegant solution but `filepath.Abs(path)` didn't add the `/private` to wd, and `os.Readlink` errors on the test temp dir.